scripts/update/Updater.php

```diff
@@ -324,5 +326,31 @@ public function update($initialVersion)
         }
 
         $this->skip('1.10.2', '1.13.2');
+
+        if ($this->isVersion('1.13.2')) {
+
+            $storageService  = $this->getServiceManager()->get(Storage::SERVICE_ID);
+
+            if ($storageService instanceof Sql && ! $storageService instanceof PaginatedStorage) {
```

Comment:
You can use only compatibilityChecker with SQL storage. But if you want diagnostic list you need paginatedStorage but it's not required